javascript/index.mjs

```diff
@@ -1,15 +1,16 @@
-module.exports = {
-  env: {
-    browser: true,
-    es2021: true
+import globals from 'globals'
+
+export const CacheESLintConfig = {
+  languageOptions: {
+    globals: {
+      ...globals.node,
+      ...globals.browser,
```

Comment:
Wouldn't it make more sense to merge all of globals in? Not exactly sure I understand what is happening here.

Reply:
Yeah I suppose. It just lets you specify which environments/globals you want to lint for.